src/EditorFeatures/CSharpTest/Squiggles/ErrorSquiggleProducerTests.cs

```diff
@@ -49,7 +49,7 @@ public async Task ErrorTagGeneratedForError()
             Assert.Equal(PredefinedErrorTypeNames.SyntaxError, firstSpan.Tag.ErrorType);
         }
 
-        [WpfFact, Trait(Traits.Feature, Traits.Features.ErrorSquiggles)]
+        [WpfFact(Skip = "https://github.com/dotnet/roslyn/issues/46463"), Trait(Traits.Feature, Traits.Features.ErrorSquiggles)]
```

Comment:
still need to skip?  the linked issue is closed.

Reply:
Removed.

Reply:
still seems relevant.